src/common/appId.js

```diff
@@ -35,3 +33,68 @@ export const oauthLogin = (done = () => 0) => {
         done();
     }
 };
+
+export const getAppIdFallback = () => getStorage('config.app_id') || getDefaultEndpoint().appId;
+
+export const getDefaultEndpoint = () => ({
+    url  : 'frontend.binaryws.com',
+    appId: getStorage('config.default_app_id') || 1169,
+});
+
+export const getWebSocketURL = () =>
+    `wss://${getStorage('config.server_url') || getDefaultEndpoint().url}/websockets/v3`;
+
+export const getOAuthURL = () =>
+    `https://${getStorage('config.server_url') || getDefaultEndpoint().url}/oauth2/authorize?app_id=${getStorage(
+        'config.app_id'
+    ) || getDefaultEndpoint().appId}&l=${getLanguage().toUpperCase()}`;
+
+const options = {
+    apiUrl   : getWebSocketURL(),
+    websocket: typeof WebSocket === 'undefined' ? require('ws') : undefined, // eslint-disable-line global-require
+    language : getStorage('lang') || 'en',
+    appId    : getAppIdFallback(),
+};
+
+const addTokenIfValid = token =>
```

Comment:
Thanks for bringing these functions out of common-utils. Could you please bring their tests too? I think now that they're in a modern project, we can make use of `async/await` to make them readable, but this can be done on a separate card.